### DISCUSSION TITLE
Drag and drop the installation binary into this terminal window and press Enter:

### DISCUSSION THREAD

**Discussion by stephen12726** [ichi4096/vivado-on-silicon-mac#36] [Q&A]
I have questions in step 3 of Installation. After I finished the step 1 and 2, my Mac Terminal told me to "Drag and drop the installation binary into this terminal window and press Enter:"  But it just stop here. which files should I drag and drop in or what should I do to keep going? Thanks.

**Comment by madsamtoft** [comment]
I tried searching around, and i believe it is the ".bin" file that you need to download from the xilinx website (the "Linux self extracting web installer"). Still can't get it to work, but i will post the solution if i find it

**Comment by ichi4096** [comment]
It is the Linux self extracting web installer. You need to download it and put it in the "vivado-on-silicon-mac-main" folder. Then, start the script and drag and drop the installation file into the terminal window and press enter.
